lib/ITip/Broker.php

```diff
@@ -572,6 +583,9 @@ protected function parseEventForOrganizer(VCalendar $calendar, array $eventInfo,
                 foreach ($attendee['newInstances'] as $instanceId => $instanceInfo) {
 
                     $currentEvent = clone $eventInfo['instances'][$instanceId];
+
+                    $this->stripVAlarms($currentEvent);
```

Comment:
This can be shortened to:

    unset($currentEvent->VALARM);

And then you don't need that extra function